static/app/views/dashboardsV2/dashboard.tsx

```diff
@@ -262,6 +263,12 @@ class Dashboard extends Component<Props, State> {
     if (!!!isEditing) {
       handleUpdateWidgetList(nextList);
     }
+    // Force check lazyLoad elements that might have shifted into view after deleting an upper widget
+    // Unfortunately need to use setTimeout since React Grid Layout animates widgets into view when layout changes
+    // RGL doesn't provide a handler for post animation layout change
+    setTimeout(() => {
```

Comment:
```suggestion
    setTimeout(forceCheck, 400);
```

Really small thing but we can inline this, clears up a bit of the visual clutter with the parens and braces